src/ByteSizeLib/ByteSizeLib.csproj

```diff
@@ -28,6 +28,8 @@ View all release notes at https://github.com/omar/ByteSize/releases.</PackageRel
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
```

Comment:
Don't you think it would make sense to keep the warnings so you can document whatever is missing today?

I believe it is a best practice to have full documentation for libraries like this.

Reply:
Looks like I had "TreatWarningsAsErrors" in the config which failed the build. I'll push a change to remove this <NoWarn> and the <TreatWarningsAsErrors>.

![image](https://user-images.githubusercontent.com/196368/140431157-6bcd721f-d7f3-4c8a-b235-b6ca40ebc2c0.png)